pkg/controller/nicclusterpolicy/nicclusterpolicy_controller.go

```diff
@@ -124,6 +128,11 @@ func (r *ReconcileNicClusterPolicy) Reconcile(request reconcile.Request) (reconc
 		return reconcile.Result{}, err
 	}
 
+	if !isSupportedInstance {
```

Comment:
can you move L114 here ? it would be clearer as this is the only place that you use it.
better yet, just 

```
// Support only a single instance of NicClusterPolicy
if request.Name != consts.NicClusterPolicyResource name {
....
}
``